config/prod.exs

```diff
@@ -15,50 +15,22 @@ use Mix.Config
 # which you typically run after static files are built.
 config :advisor, Advisor.Web.Endpoint,
   on_init: {Advisor.Web.Endpoint, :load_from_system_env, []},
-  url: [host: "example.com", port: 80],
-  cache_static_manifest: "priv/static/cache_manifest.json"
+  url: [scheme: "https", host: "advisorex.herokuapp.com", port: 443],
+  force_ssl: [rewrite_on: [:x_forwarded_proto]],
+  cache_static_manifest: "priv/static/cache_manifest.json",
+  secret_key_base: System.get_env("SECRET_KEY_BASE")
 
 # Do not print debug messages in production
```

Comment:
Yes, that is the way it is configured at the moment. 
There are 3 config files: `prod.exs`, `dev.exs`, and `test.exs`. I believe dev prints, but test doesn't.